chain/westend_dev/defaults.go

```diff
@@ -1,11 +1,9 @@
 // Copyright 2021 ChainSafe Systems (ON)
 // SPDX-License-Identifier: LGPL-3.0-only
 
-package gssmr
+package westend_dev
```

Comment:
nit: No underscores in package name

Reply:
Why are we using the underscores at all as opposed to hyphens?